src/handlers/cell.ts

```diff
@@ -238,3 +250,12 @@ export interface ILineInfo {
   /** Array of line widgets attached to this line. */
   widgets: any;
 }
+
+namespace Private {
+  export function createMarkerNode() {
```

Comment:
Do we need to export this function?

Reply:
The function is exported but the namespace is not.